src/BookingForm.js

```diff
@@ -0,0 +1,195 @@
+import React, { useState } from "react";
+import SearchResults from "./SearchResults.js";
+import TableRow from "./TableRow.js";
+function BookingForm({ handleNewBooking }) {
+  const [isSubmitClicked, setSubmitClicked] = useState(false);
+  const [newBookingInfo, setNewbookingInfo] = useState({
+    id: "0",
+    title: "",
+    firstName: "",
+    surname: "",
+    email: "",
+    roomId: "",
+    checkInDate: "",
+    checkOutDate: "",
+    items: []
+  });
+
+  function handleSelect(event) {
+    let newId = 5;
+    newId++;
```

Comment:
`newId` will always be fixed as 6 as the value will be reset on every call and doesn't persist in state